core/offchain.go

```diff
@@ -187,13 +186,12 @@ func (bc *BlockChain) CommitOffChainData(
 		for i, c := uint32(0), shard.Schedule.InstanceForEpoch(
 			epoch,
 		).NumShards(); i < c; i++ {
-			if err := bc.LastContinuousCrossLink(batch, i); err != nil {
-				utils.Logger().Info().
-					Err(err).Msg("could not batch process last continuous crosslink")
-			}
+			bc.LastContinuousCrossLink(batch, i)
```

Comment:
if its not even worth logging out an error, then remove the error return value

Reply:
it may be useful for other cases. Noted though.

Reply:
then keep the error value and leave a NOTE , remove the log. leaving the error unhandled code with no comemnt explaining is that was intentional or not is ambigious for tomorrrow.

Reply:
https://github.com/harmony-one/harmony/issues/2762. Don't want to restart the jenkins.